assets/queries/openAPI/operation_without_successful_http_status_code/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	responses := doc.paths[n][oper].responses
+
+	count({x | response := responses[x]; regex.match(`^2[0-9]{2}$`, x) == true}) == 0
```

Comment:
I think this regex code should check only the following codes: 200, 201, 202 and 204, which are specific for Rest APIs, as you can see [in this resource on "REST Specific Status Codes" section](https://restfulapi.net/http-status-codes/).
Otherwise, if want to check all valid successful response codes, should be 200, 201, 202, 203, 204, 205, 206, 207, 208 and 226